### ISSUE TITLE
A11y_VSCode_ChangingtheUsersettings_ScreenReader: Screen reader does not announce expanded, collapsed state of 'Editor: Accessibility Support' combo box when we expand or collapse it.

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode#76945]
**Environment Details:**
VSCode Version : 1.36.1 (user setup)
OS Version : MacOS Mojave 10.14.5

**Additional Details:**
MAS Violated : MAS 1.3.1
Screen Reader Versions : Voice over

**Repro Steps:**
1.Open Visual studio code editor.
2.Open the command Pallet by pressing "F1" key and type "setting".
3.Using down-arrow key select user setting.
4.Type the word "Auto " and select one of the suggestion given by the vs code editor.
5.Type the "Tab size" and select the option from the auto suggestion.
6.Start screen reader

**Actual:**
Screen reader does not announce expanded, collapsed state of 'Editor: Accessibility Support' combo box when we expand or collapse it.

This issue is observed in Mac + Voice over

**Expected:**
Screen reader should announce the expanded, collapsed state when we expand or collapse the 'Editor: Accessibility Support' combo box

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not understand the combo box is actually expanded or not. 

**MAS Reference:**
**MAS 1.3.1 -** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_ChangingtheUsersettings_ScreenReader_Mac.pptx](https://github.com/microsoft/vscode/files/3371873/A11y_VSCode_ChangingtheUsersettings_ScreenReader_Mac.pptx)

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/76869
